cocos2d/core/components/CCStudioComponent.js

```diff
@@ -288,6 +288,7 @@ var StudioComponent = cc.Class({
     },
 });
 
+// TODO: nest by the built-in manner
```

Comment:
@knoxHuang We should use synced prefab to implement this component. Or just discard it since the editor now supports nested prefab.